extensions/Webhook.cpp

```diff
@@ -0,0 +1,53 @@
+// PabstMirror
+// async send msg to a discord webhook
+// Requires CPR (curl wrapper) - https://github.com/whoshuu/cpr (and put libcurl.dll in base arma folder)
+
+#include <iostream>
+#include <String>
+#include <future>
+#include "cpr/cpr.h"
+
+#define VERSION_STR "v1.0.0"
+
+
+extern "C" {
+    __declspec(dllexport) void __stdcall RVExtension(char* output, int outputSize, const char* function);
+    __declspec(dllexport) void __stdcall RVExtensionVersion(char* output, int outputSize);
+    __declspec (dllexport) void __stdcall RVExtensionRegisterCallback(int(*callbackProc)(char const* name, char const* function, char const* data));
+}
+
+int(*callbackPtr)(char const* name, char const* function, char const* data) = nullptr;
```

Comment:
couldn't `std::function<int(char const*, char const*, char const*)> callBackPtr = []() -> int { return 0; }` work instead of a function pointer?